circom-mpc-compiler/src/lib.rs

```diff
@@ -142,15 +173,19 @@ where
         program_archive: ProgramArchive,
     ) -> Result<(CircomCircuit, OutputMapping)> {
         let build_config = BuildConfig {
-            no_rounds: usize::MAX, //simplification_style. Use default from their lib
+            no_rounds: if let Simplification::O2(r) = self.config.simplification {
+                r
+            } else {
+                0
+            },
             flag_json_sub: false,
             json_substitutions: String::new(),
-            flag_s: false,
-            flag_f: false,
+            flag_s: self.config.simplification == Simplification::O1,
+            flag_f: self.config.simplification == Simplification::O0,
```

Comment:
Are these really exclusive? What happens if both are `true`?

Reply:
They are exclusive in circom (https://github.com/iden3/circom/blob/master/circom/src/input_user.rs#L100C1-L102C77)
If both are passed as args in circom, the "lower" (in this case 0) one will be be used, for co-cricom only one can be set in the config